### PR TITLE
updating Node example app documentation

### PR DESCRIPTION
**Pull request by mansona**
I wasn't sure if you wanted to replace the current documentation for a Node example, I only did because it was marked as "outdated". 

Let me know if you want me to leave it in :+1:

### CODE REVIEW

**Comment by lukemelia** [comment]
LGTM. @ghedamat?

**Comment by ghedamat** [comment]
@mansona going trough the commits it seems [ember-lightning](https://github.com/philipheinser/ember-lightning/commits/master) has been updated to work with 0.5.X

might be fair to keep both the links

LGTM otherwise

thanks!

File: docs/v0.5.x/lightning-strategy-examples.md [hunk 1]
```diff
@@ -40,11 +40,13 @@ end
 
 The nice thing about this is that you can deploy your app to production, test it out by passing an index_key parameter with the revision you want to test and activate when you feel confident that everything is working as expected.
 
-## Example Node app (outdated)
+## Example Node app
 
-This app does the same as the *Sinatra app* above, it supports the same index_key query param. It should help you to get up and running in seconds and dont worry about server code. You also need to deploy your ember app with `ember-cli-deploy` and `ember-deploy-redis`.
```
Comment (ghedamat) [inline docs/v0.5.x/lightning-strategy-examples.md]: ok this mention of `ember-deploy-redis` is wrong.

maybe keep your description, rename the section to `example node apps` and leave at the end the link to ember-lightning?

**Comment by mansona** [comment]
@ghedamat awesome, I'll put it back in an differentiate them based on Node framework so :+1:

File: docs/v0.5.x/lightning-strategy-examples.md (hunk 1 above)
Comment (mansona) [inline docs/v0.5.x/lightning-strategy-examples.md]: So I've updated my side to have the correct reference to `ember-cli-deploy-lightning-pack` and I'll update the previous reference when I'm incorporating it back in :+1:

**Comment by mansona** [comment]
@ghedamat I've added the previous docs back in and updated their reference to ember-cli-deploy-lightning-pack. So this should be good to go :tada: :smile:

**Comment by ghedamat** [comment]
@mansona perfect!

Last request  could you squash into one commit?

**Comment by mansona** [comment]
@ghedamat cool that's done now :+1: that's my first squash so it took a bit of googling :wink:

**Comment by ghedamat** [comment]
@mansona thanks a lot!

**Pull request merged**